src/run/run.c

```diff
@@ -662,6 +669,20 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
                 }
 
+                case ARG_ROOT_DIRECTORY:
+                        r = parse_path_argument(optarg, /* suppress_root= */ true, &arg_root_directory);
```

Comment:
Should we suppress root here? The point of the switch is that the root of the execution env can differ from that of pid1?